harmonica/tests/test_point_gravity.py

```diff
@@ -551,19 +563,19 @@ def tensor_finite_differences(coordinates, point, mass, field, delta=0.05):
     # computed
     direction_i, direction_j = field[-2], field[-1]
     if direction_i == "e":
-        direction_i = "easting"
+        direction_i = "e"
     if direction_i == "n":
-        direction_i = "northing"
+        direction_i = "n"
     if direction_j == "e":
-        direction_j = "easting"
+        direction_j = "e"
     if direction_j == "n":
-        direction_j = "northing"
+        direction_j = "n"
```

Comment:
No we don't need these lines 😀, feel free to ditch them

Reply:
> No we don't need these lines 😀, feel free to ditch them

Haha yeah~ Don't know why I keep them in here 😅